packages/editor/src/components/post-card-panel/index.js

```diff
@@ -88,8 +88,8 @@ export default function PostCardPanel( { className, actions, children } ) {
 							className="editor-post-card-panel__description"
 							spacing={ 2 }
 						>
-							{ !! description && <Text>{ description }</Text> }
-							{ !! lastEditedText && (
+							{ description && <Text>{ description }</Text> }
+							{ lastEditedText && (
```

Comment:
Nice. These are definitely not numbers and `0`s won't be displayed, so no need for the boolean conversion.